solutions/swb-reference/src/SWBStack.ts

```diff
@@ -861,7 +847,7 @@ export class SWBStack extends Stack {
       gsis: [
         {
           indexName: 'getIdentityPermissionsByIdentity',
-          partitionKey: { name: 'Identity', type: AttributeType.STRING },
+          partitionKey: { name: 'identity', type: AttributeType.STRING },
```

Comment:
What does the change in capitalization do?

Reply:
@htztomic could you help pointing out the external change for this?